CONTENT/schedule.markdown

```diff
@@ -38,11 +38,11 @@ We currently organize 2 sessions per week. Every magic session accepts **up to 1
 <img alt="" border="0" src="https://www.paypalobjects.com/en_US/i/scr/pixel.gif" width="1" height="1">
 </form>
 
-**If you can't afford to pay the full price, we are proud to be able to offer you our Pay-what-you-can program, supported by [our amazing donors](...).**
+**If you can't afford to pay the full price, we also offer Pay-what-you-can program, supported by [our amazing donors](...).**
 
-**[=> APPLY](mailto:wizard@amigos.institute?Subject=Application:%20For%20the%20Pay-what-you-can%20program&Body=%0D%0A)**
+**[=> APPLY FOR PAY-WHAT-YOU-CAN PROGRAM](mailto:wizard@amigos.institute?Subject=Application:%20For%20the%20Pay-what-you-can%20program&Body=%0D%0A)**
 
-*If you have any further questions, just [drop us a mail](mailto:wizard@amigos.institute) or make use of your free trial session and check us out :-)*
+*If you have any further questions, just [drop us an e-mail](mailto:wizard@amigos.institute) and/or come to our free trial session :)*
```

Comment:
Ok, it's shorter. Would we like them to write us an email before they come to our school sessions?
If so, maybe the "or" should be removed, but if you really think both options are good, then: +1